iac-extensions/spring-config/src/main/java/org/sonar/iac/springconfig/plugin/SpringConfigSettings.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.springconfig.plugin;
+
+import java.util.List;
+import org.sonar.api.PropertyType;
+import org.sonar.api.config.PropertyDefinition;
+import org.sonar.api.resources.Qualifiers;
+
+public final class SpringConfigSettings {
+  static final String ACTIVATION_KEY = "sonar.springconfig.activate";
+  static final String ACTIVATION_DEFAULT_VALUE = "true";
+  static final String FILE_PATTERNS_KEY = "sonar.springconfig.file.patterns";
+  static final String DEFAULT_FILE_PATTERNS = "**/src/main/resources/**/application*.properties," +
```

Comment:
1. it can be private
2. please be consisted with names, e.g. `FILE_PATTERNS_DEFAULT_VALUE` will follow a pattern similar to `ACTIVATION_DEFAULT_VALUE`